torch/csrc/jit/passes/guard_elimination.cpp

```diff
@@ -273,6 +273,20 @@ struct GuardElimination {
     case aten::reciprocal:
     case aten::addcmul:
     case aten::where:
+    case aten::softmax:
+    case aten::log_softmax:
+    case aten::log_sigmoid:
+    case aten::dropout:
+    case aten::matmul:
+    case aten::bmm:
+    case aten::_cast_Long:
+    case aten::_cast_Short:
+    case aten::_cast_Int:
+    case aten::_cast_Float:
+    case aten::_cast_Half:
+    case aten::_cast_Double:
+    case aten::contiguous:
```

Comment:
i can't seem to find a lot of what `aten::contiguous` actually does?

Reply:
I'm pretty sure it forces the tensor to be contiguous without changing the shape or value.